openapi/nexus.json

```diff
@@ -6883,6 +6980,33 @@
         }
       }
     },
+    "/v1/utilization": {
```

Comment:
Should this not be `/v1/system/utilization`? It also has the tag "silos" like the other endpoints. Conversely, if we want to keep this one as `/v1/utilization`, perhaps we could change the other endpoints to be `/v1/system/utilization`. Otherwise it feels like one set of endpoints is very explicit and the other is not. WDYT?

Reply:
This is the silo admin's view of the utilization inside the silo. So this is a developers way of seeing how many resources their silo can provision. I dropped `/v1/system/utilization` (which summarizes the entirety of the system utilization) from this PR as it was more complicated to deliver. 

In my mind `/v1/system/...` means operator whereas `/v1/...`

Reply:
> I dropped /v1/system/utilization (which summarizes the entirety of the system utilization) from this PR as it was more complicated to deliver.

Ah! I think this was the piece of information I was missing. If we want to have this endpoint in the future then the current implementation makes sense, thanks!